nxcomp/configure.in

```diff
@@ -1,413 +0,0 @@
-  else
-    if test -f "/usr/lib/libjpeg.a" ; then
-      LIBSTATIC="$LIBSTATIC /usr/lib/libjpeg.a"
-    else
-      if test -f "/usr/local/lib/libjpeg.a" ; then
-        echo -e "assuming libjpeg.a in /usr/local/lib"
-        LIBSTATIC="$LIBSTATIC /usr/local/lib/libjpeg.a"
-      else
-        echo -e "Warning: assuming libjpeg.a in the local path"
-        LIBSTATIC="$LIBSTATIC libjpeg.a"
-      fi
-    fi
-  fi
-else
-  echo -e "enabling dynamic linking of JPEG library"
-  LIBSHARED="$LIBSHARED -ljpeg"
-fi
-
-dnl Check whether --with-static-z was given and
-dnl add -lz or libz.a to linking.
-
-if test "${with_static_z}" = yes; then
-  echo -e "enabling static linking of Z library"
-  if test "$SUN" = yes && test -f "/usr/sfw/lib/libz.a"; then
-    LIBSTATIC="$LIBSTATIC /usr/sfw/lib/libz.a"
-  else
-    if test -f "/usr/lib/libz.a" ; then
-      LIBSTATIC="$LIBSTATIC /usr/lib/libz.a"
-    else
-      if test -f "/usr/local/lib/libz.a" ; then
-        echo -e "assuming libz.a in /usr/local/lib"
-        LIBSTATIC="$LIBSTATIC /usr/local/lib/libz.a"
-      else
-        echo -e "Warning: assuming libz.a in the local path"
-        LIBSTATIC="$LIBSTATIC libz.a"
-      fi
-    fi
-  fi
-else
-  echo -e "enabling dynamic linking of Z library"
-  LIBSHARED="$LIBSHARED -lz"
-fi
-
-dnl Finally compose the LIB variable.
-
-if test "$DARWIN" = yes ; then
-  LIBS="$LIBS $LIBSTATIC $LIBSHARED"
-elif test "$SUN" = yes ; then
-  LIBS="$LIBS $LIBSTATIC $LIBSHARED"
-else
-  LIBS="$LIBS $LIBSTATIC -shared $LIBSHARED"
-fi
-
-dnl Check whether --with-symbols or --without-symbols was
-dnl given and set the required optimization level.
-
-if test "${with_symbols}" = yes; then
-  echo -e "enabling production of debug symbols"
-  CXXFLAGS="-g $CXXFLAGS"
-  CFLAGS="-g $CFLAGS"
-else
-  echo -e "disabling production of debug symbols"
-fi
-
-dnl Check whether --with-use-malloc or --without-use-malloc
-dnl was given.
-
-if test "${with_use_malloc}" = yes; then
-  echo -e "disabling use of the STL allocators"
-  CXXFLAGS="$CXXFLAGS -D__USE_MALLOC"
-else
-  echo -e "enabling use of the STL allocators"
-fi
-
-dnl Check whether --with-info or --without-info was given.
-
-if test "${with_info}" = yes; then
-  echo -e "enabling info output in the log file"
-  CXXFLAGS="$CXXFLAGS -DINFO"
-  CFLAGS="$CFLAGS -DINFO"
-else
-  echo -e "disabling info output in the log file"
-fi
-
-dnl Check whether --with-valgrind or --without-valgrind was given.
-
-if test "${with_valgrind}" = yes; then
-  echo -e "enabling valgrind memory checker workarounds"
-  CXXFLAGS="$CXXFLAGS -DVALGRIND"
-  CFLAGS="$CFLAGS -DVALGRIND"
-else
-  echo -e "disabling valgrind memory checker workarounds"
-fi
-
-dnl Find makedepend somewhere.
-
-AC_SUBST(MAKEDEPEND)
-MAKEDEPEND="$(which makedepend)"
-
-dnl Determine what to build based on the platform.
-dnl Override the LIBS settings on Cygwin32 so that
-dnl we always link with the exact set of libraries.
-
-AC_SUBST(ALL)
-
-if test "$CYGWIN32" = yes; then
```

Comment:
> Special handling for cygwin that gets lost... I figure you had no chance to try Cygwin-based builds?

I plan to contact mikedep333 on that. I guess, he will be much faster than us providing the required info, if any are needed. Leave it as is for now.